File: TenMinsJava/day37.java
// get remaining microseconds of the current day
package TenMinsJava;

import java.time.LocalTime;

public class day37 {
    public static void main(String[] args) {
        LocalTime t = LocalTime.now();
        long micro = t.toNanoOfDay() / 1000;
        System.out.println(86400000000L - micro);
    }
}

File: TenMinsPy/day37.py
# get remaining microseconds of current the day

from datetime import datetime

class day36:
    def __init__(self):
        n = (datetime.now().hour * 3600000000) + (datetime.now().minute * 60000000) + (datetime.now().second * 1000000) + datetime.now().microsecond
        print(86400000000 - n)

day36()